htdocs/core/tpl/list_print_total.tpl.php

```diff
@@ -13,7 +13,7 @@
 	while ($i < $totalarray['nbfield']) {
 		$i++;
 		if (!empty($totalarray['pos'][$i])) {
-			switch ($totalarray['type'][$i]) {
+			switch ($totalarray['pos'][$i]) {
```

Comment:
The different cases are different types. But you replace the variable of type with position ? Why ?